ynput/ayon-zbrush
# Add .gitignore and move the zscript menu into temp folder

## Changelog Description
This PR is the continuity of https://github.com/ynput/ayon-zbrush/pull/15
- [ ] move the zscript menu to temp
## Additional info
n/a

## Testing notes:

1. Launch Zbrush via Ayon
2. Ayon menu should appear
3. All Ayon Plugins work
4. Close Zbrush
